flutter_bird_image_server/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const cors = require('cors');
+const { mintRandomSkin } = require("./create_birds/create_data");
+
+const app = express();
+
+app.use(cors({
+  origin: '*',
+  credentials: true,
+  optionsSuccessStatus: 200
+}));
+
+app.get('/api/image/:tokenId', async (req, res) => {
+  try {
+    const tokenId = req.params.tokenId;
+    const dataUrl = await mintRandomSkin(tokenId);
+    res.json({ imageDataUrl: dataUrl });
+  } catch(e) {
+    console.error('Error in /api/image/:tokenId route:', e);
+    res.status(500).json({ error: 'Internal server error' });
+  }
+});
+
+app.get('/', (req, res) => {
+  res.status(200).json({ status: 'OK', message: 'Server is running' });
+});
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
What about for production?

Reply:
This only needs to be executed when you want to specify a port when starting locally, so currently `process.env.NODE_ENV === 'production'` is not necessary.
(When deployed to vercel, process.env.NODE_ENV will be production.)